Presentation/UIViewController+Signals.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  UIViewController+Signals.swift
+//  Presentation
+//
+//  Created by João D. Moreira on 2018-11-02.
+//  Copyright © 2018 iZettle. All rights reserved.
+//
+
+import Foundation
+import Flow
+import UIKit
+
+public extension UIViewController {
+    var viewDidLoadSignal: Signal<Void> {
+        return Signal(callbacker: callbackers.viewDidLoad)
+    }
+
+    var viewWillAppearSignal: Signal<Bool> {
+        return Signal(callbacker: callbackers.viewWillAppear)
+    }
+
```

Comment:
Looks like `viewDidAppear` is missing :) There is of course also the callback `didReceiveMemoryWarning`, but maybe it doesn't belong here.

Reply:
Good catch. I will add `viewDidAppear`. Not sure about `didReceiveMemoryWarning`, maybe worth it?

Reply:
Low memory can be handled via notifications instead:

```swift
bag += NotificationCenter.default.signal(forName: UIApplication.didReceiveMemoryWarningNotification).onValue { ... }
```